tests/test_sections.cfg

```diff
@@ -28,5 +28,3 @@ active = False
 #[mpas_analysis]
 #active = False
 
```

Comment:
This blank line needs to be removed to pass the `pre-commit run --all-files` checks.